submit/main/notify.py

```diff
@@ -59,33 +63,57 @@ def prepare_upload(file: FileStorage):
 
 
 def send_confirmation_emails(
-    excel_file: FileStorage, fund: str, reporting_period: str, fund_email: str, user_email: str, metadata: dict
-):
-    """Sends a confirmation email to the LA that submitted and to the Fund Team (with a link to the submission).
+    excel_file: FileStorage,
+    fund: FundConfig,
+    user_email: str,
+    metadata: dict,
+) -> None:
+    """Sends a confirmation email to the LA that submitted and to the Fund Team
+    with a link to the submission.
 
     :param excel_file: Excel file that has been submitted successfully
-    :param fund: the fund name for the submission
-    :param reporting_period: the reporting period for the submission
-    :param fund_email: the fund's email address to send confirmation to
+    :param fund: FundConfig object for the fund that the submission was made to
     :param user_email: the user's email address to send confirmation to
     :param metadata: contains information about the submission
+
     :return: None
     """
-    personalisation = get_personalisation(excel_file, fund, reporting_period, metadata)
+
+    personalisation = get_personalisation(excel_file, fund, metadata)
+
     current_app.logger.info("Sending confirmation emails to LA and Fund Team")
-    # to the Local Authority
-    send_email(email_address=user_email, template_id=Config.LA_CONFIRMATION_EMAIL_TEMPLATE_ID, **personalisation)
-    # to Fund - includes the file
+
+    # email Local Authority
     send_email(
-        email_address=fund_email,
+        email_address=user_email,
+        template_id=Config.LA_CONFIRMATION_EMAIL_TEMPLATE_ID,
+        **personalisation,
+    )
+
+    programme_id = metadata.get("Programme ID")
+    round_number = fund.current_reporting_round
+
+    programme = get_programme_by_id_and_round(programme_id, round_number)
+    _, submission_id = get_or_generate_submission_id(programme, round_number, fund.fund_code)
+
+    object_name = f"{fund.fund_code}/{submission_id}"
+    personalisation.update(
+        {
+            "download_url": f"{Config.FIND_SERVICE_BASE_URL}/retrieve-spreadsheet/{object_name}",
+        }
+    )
+
+    # email to Fund
+    send_email(
+        email_address=fund.email,
         template_id=Config.FUND_CONFIRMATION_EMAIL_TEMPLATE_ID,
-        file=excel_file,
+        notify_key=Config.NOTIFY_FIND_API_KEY,
```

Comment:
This doesn't feel right for a couple of reasons:

* Code in the `submit` module is now using the Find API key.
* These emails have always come via the Submit service before - why do we want to send them from the Find service now? It looks like you've made the new template in the Find service too - how come?

Reply:
`notify.py` lives in `data_store.controllers`

Reply:
Good point on the template.. I created it on Find. But retrieve-download is a Fine related feature. 

Perhaps template should be on Submit? :thinking:

Reply:
Doesn't overlay matter I guess... Current template is on submit so let's keep it there.

Reply:
Moved it to submit.

Reply:
Is there any reason we need a new template at all...?

Also just looked at the template - it says the link will stop working after 7 days. That's not true? I think we could have just reused the existing email, not sure why we need a new template.

Reply:
Find email claims the link expires in 7 days. Are the two different?

Reply:
Yes. The find bucket has a 7 day lifecycle policy on it. The submitted spreadsheets bucket doesn't, I believe - maybe check.

Reply:
Ooo gross, you're right. I'll get on and fix that 7 days thing for find downloads 👀

Reply:
Fixed that 👍

Reply:
I managed to switch to the old template.. I thought that template would be missing some parts we need. But it does include a download link.

@Kateharries is it ok if we stick to it?
![image](https://github.com/user-attachments/assets/0d3e3bf5-3cb7-4aa5-a90f-a6032af83883)